tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -390,6 +390,39 @@ void ModularFileSystem::FlushCaches(TransactionToken* token) {
   if (ops_->flush_caches != nullptr) ops_->flush_caches(filesystem_.get());
 }
 
+Status ModularFileSystem::SetConfiguration(const std::string& name,
+                                           const std::vector<string>& values) {
+  if (ops_->set_filesystem_configuration == nullptr) {
+    return errors::Unimplemented(
+        "Filesystem does not support SetConfiguration()");
+  }
+  if (values.size() == 0) {
+    return errors::InvalidArgument(
+        "SetConfiguration() needs number of values > 0");
+  }
+
+  TF_Filesystem_Option option;
+  memset(&option, 0, sizeof(option));
+  option.name = const_cast<char*>(name.c_str());
+  option.per_file = 0;
```

Comment:
This is not needed, given the memset above.

Reply:
Thanks @mihaimaruseac . Removed.